internal/services/serviceconnector/service_connector_app_service_resource_test.go

```diff
@@ -132,6 +132,13 @@ resource "azurerm_linux_web_app" "test" {
   service_plan_id     = azurerm_service_plan.test.id
 
   site_config {}
+
+  lifecycle {
+    ignore_changes = [
+      app_settings["AZURE_STORAGEBLOB_RESOURCEENDPOINT"],
+      identity,
+    ]
+  }
```

Comment:
We shouldn't ever ignore identity, could you provide some more information on the failure?

Reply:
Sure. We're ignoring identity because `azurerm_app_service_connection` resource is using `systemAssignedIdentity`, this will pass the credentials to `azurerm_linux_web_app`, so it will cause diff on this `azurerm_linux_web_app` resource. 

```hcl
 Terraform will perform the following actions:
        
          # azurerm_linux_web_app.test will be updated in-place
          ~ resource "azurerm_linux_web_app" "test" {
                id                                = "/subscriptions/85b3dbca-5974-4067-9669-67a141095a76/resourceGroups/acctestRG-230227101737183561/providers/Microsoft.Web/sites/linuxwebappro1bv"
                name                              = "linuxwebappro1bv"
                tags                              = {}
                # (18 unchanged attributes hidden)
        
              - identity {
                  - identity_ids = [] -> null
                  - principal_id = "80cf7994-1c40-4405-962e-592a8e9065e3" -> null
                  - tenant_id    = "72f988bf-86f1-41af-91ab-2d7cd011db47" -> null
                  - type         = "SystemAssigned" -> null
                }
        
                # (2 unchanged blocks hidden)
            }
        
        Plan: 0 to add, 1 to change, 0 to destroy.
```

Reply:
this sounds like a bug in the resources? these properties should become computed?

Reply:
If the service connection resource is modifying the web app outside normal methods, then `ignore_changes` on `identity` is unfortunately going to be the the correct way. This resource is not actually part of the Web / App Service RP and belongs to `Service Linker` so is not modifying the resource by any mechanism we can account for. Users will also need to make this lifecycle change when using the resource to link an App to a `target_resource_id`, this should be added to the Documentation also for the Service Connector resource.

Reply:
Sure. I'll update the documentation accordingly.

Reply:
alternatively users could add the System Assigned Identity to the Linux Web App too, which would solve that issue?

Reply:
Yes. That requires the user to set the identity type as `systemAssigned`. I'll specify this in the doc also.